setup.py

```diff
@@ -1,18 +1,43 @@
 #!/usr/bin/python3
 import setuptools
+import sys
```

Comment:
I usually use `from sys import argv`, but whatever floats your boat I guess

Reply:
I think it's better this way, because then you import the namespace and it's less likely to get naming conflicts